diff -ur rubberband.pc.in.bak rubberband.pc.in
--- rubberband.pc.in.bak	2017-05-21 23:35:23.515625000 +0200
+++ rubberband.pc.in	2017-05-23 23:05:01.156250000 +0200
@@ -5,6 +5,7 @@
 
 Name: rubberband
 Version: 1.8.2
-Description: 
+Description: An audio time-stretching and pitch-shifting library and utility program
 Libs: -L${libdir} -lrubberband
+Libs.private: %LIBRARY_LIBS%
 Cflags: -I${includedir} 
